types/gifencoder/gifencoder-tests.ts

```diff
@@ -0,0 +1,43 @@
+import GIFEncoder = require('gifencoder');
+import fs = require('fs');
+
+declare const pngFileStream: NodeJS.ReadableStream;
```

Comment:
just btw, these tests are just for linting and they are not executed really, 
so there is no need for a tricks like that one and there could be a proper code 
here that obtains a stream  - just like in the real life scenario.
It won't be executed, just run through compiler/linter checks

Reply:
I know, but the original example depended on the `png-file-stream` package and so I just decided to replace that with a `declare const`. Should I change it?